### DISCUSSION TITLE
Add catalog item for mapbox raster tiles

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#5066] [Q&A] [answered]
Can anyone provide an example for adding a catalog item for mapbox raster tiles served from TileServer-GL or GeoServer?
The current url to get this data into leaflet is https://my-maps.my-server.org/styles/osm-bright/{z}/{x}/{y}.png
Thank you!

**Comment by geospatialmapmashup** [comment, marked as answer]
rename attached .json
[Catalog-08.12.2020 14_10_29.txt](https://github.com/TerriaJS/terriajs/files/5661315/Catalog-08.12.2020.14_10_29.txt)

URL Template is all you need or Open Street map and remove placeholders

**Reply by lstillwe** [reply]
Thank you!
